dashboard/test/ui/test_status.js

```diff
@@ -328,3 +320,13 @@ autoRefreshButton.onclick = toggleAutoRefresh;
 updateProgressNow();
 enableAutoRefresh();
 refresh();
+
+var hideSucceeded = false;
+function toggleHideSucceeded() {
+  hideSucceeded = !hideSucceeded;
+  hideSucceededButton.textContent = `${hideSucceeded ? 'Show' : 'Hide'} Succeeded`;
+  var sheet = document.styleSheets[document.styleSheets.length - 1];
+  sheet.insertRule(`.SUCCEEDED{display: ${hideSucceeded ? 'none' : 'inherit'}}`,sheet.cssRules.length)
```

Comment:
Do we just keep adding rules if we toggle the button over and over; that is, should we be removing them at some point?